yafti/plugin/run.py

```diff
@@ -59,6 +61,15 @@
 class Run(YaftiPlugin):
     async def exec(self, cmd: str) -> subprocess.CompletedProcess:
         log.debug("running command", cmd=cmd)
+
+        # spawn command in host when running from container
+        is_container = os.path.isfile("/run/.containerenv") or os.path.isfile("/.dockerenv")
+        if not os.path.isfile(cmd) and is_container:
+            if which("distrobox-host-exec"):
+                cmd = "distrobox-host-exec {cmd}".format(cmd=cmd)
+            elif which("flatpak-spawn"):
+                cmd = "flatpak-spawn --host {cmd}".format(cmd=cmd)
```

Comment:
NIT: as we require 3.11+ python and f strings simplify these lines
```suggestion
                cmd = f"distrobox-host-exec {cmd}"
            elif which("flatpak-spawn"):
                cmd = f"flatpak-spawn --host {cmd}"
```